Nos/Extensions/String+Markdown.swift

```diff
@@ -57,9 +56,22 @@ extension String {
             Log.error("Invalid regex pattern")
         }
         
+        replaceOccurrences(mutableString: mutableString, of: "^\\s*", with: "")
+        replaceOccurrences(mutableString: mutableString, of: "\\s*$", with: "")
+        replaceOccurrences(mutableString: mutableString, of: "\\n{2,}", with: "\n")
+        
         return (mutableString as String, urls)
     }
-
+    
+    private func replaceOccurrences(mutableString: NSMutableString, of: String, with: String) {
```

Comment:
I am ignoring the swiftlint failure in this line as I think the error is actively harmful and making me call my argument something that is different than the name of the target argument that I am passing this as.